webapi/src/main/scala/org/knora/webapi/responders/admin/ListsResponderADM.scala

```diff
@@ -2072,18 +2072,6 @@ class ListsResponderADM(responderData: ResponderData) extends Responder(responde
    * @param projectIri the IRI of the project.
    * @return a [[Boolean]].
    */
```

Comment:
Again, delete docstring as well (can this be automated?).

Reply:
no. I didn't think that it would only delete the methods and leave the docstrings. In the second batch of changes I checked every file and there removed anything that was left.